infra/src/main/java/com/redhat/service/bridge/infra/models/processors/ProcessorDefinition.java

```diff
@@ -51,6 +61,14 @@ public void setAction(BaseAction action) {
         this.action = action;
     }
 
+    public BaseAction getVirtualAction() {
```

Comment:
@kostola I'm not sure about this terminology. For me there are two Actions on the ProcessorDefinition:

1. The Action the user "requested"
2. The Action we can "resolve" their request too

These can be the same thing of course. The definition of "virtual" action is not clear to me in it's intent.

Reply:
I changed the implementation. See [here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/141#discussion_r757021840).